tycho-its/projects/TYCHO0367localRepositoryCrosstalk/bundle02/src/TYCHO0367localRepositoryCrosstalk/bundle02/Eclipse35Test.java

```diff
@@ -25,7 +25,7 @@ public void test() throws Exception {
         Bundle equinox = getBundle("org.eclipse.osgi");
 
         assertEquals(3, equinox.getVersion().getMajor());
-        assertEquals(18, equinox.getVersion().getMinor());
+        assertEquals(19, equinox.getVersion().getMinor());
```

Comment:
```suggestion
        assertTrue(equinox.getVersion().getMinor() > 13);
```

I looked in the test and it seems it only want to ensure that the 34 test is seeing an *older* version so I think we can save us some headaches by just assume the version is higher here than in the 35 test...

Reply:
Sounds right.